post_install.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+arduino_esp32_rules () {
+    echo ""
+    echo "# Arduino ESP32 bootloader mode udev rules"
+    echo ""
+cat <<EOF
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="2341", ATTRS{idProduct}=="0070", MODE:="0666"
+EOF
+}
+
+if [ "$EUID" -ne 0 ]
+  then echo "Please run as root"
+  exit
+fi
+
+arduino_esp32_rules > /etc/udev/rules.d/60-arduino-esp32.rules
+
+# reload udev rules
+echo "Reload rules..."
+udevadm trigger
+udevadm control --reload-rules
```

Comment:
Missing EOL on last line. Using a proper text editor avoids this. Unix text files should have a line terminator on every line, including the last line.